modules/lti/src/main/resources/tools/manage/js/lti.manage.js

```diff
@@ -87,6 +87,9 @@ var OCManager = (function($) {
         $('span[data-target="#scheduleModal"]').hide();
         $('span[data-target="#ttScheduleModal"]').hide();
       }
+
+      var recorderButton = document.getElementById('recorder');
+      recorderButton.href = "https://media.uct.ac.za/studio/index.html?upload.seriesId="+this.seriesId;
```

Comment:
Think this should be a host-free URL if possible otherwise testing on mediadev is going to point to media URL:


      recorderButton.href = "https://media.uct.ac.za/studio/index.html?upload.seriesId="+this.seriesId;

to

      recorderButton.href = "/studio/index.html?upload.seriesId="+this.seriesId;

Reply:
I've gone for using window.location.hostname as it seems more explicit when reading the code.
Another solution would be to do something like 
`recorderButton.href = "../../studio/index.html?upload.seriesId="+this.seriesId;`